pkgs/by-name/ga/galaxy-dummy-service/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  comet-gog,
+  meson,
+  ninja,
+  pkgsCross,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "GalaxyCommunication-dummy";
```

```suggestion
  pname = "galaxycommunication-dummy";
```
I think either this or `galaxy-dummy-service` works here.